src/static_api.rs

```diff
@@ -146,6 +147,31 @@ impl<'a> Generator<'a> {
         Ok(())
     }
 
+    fn generate_zulip_groups(&self) -> Result<(), Error> {
+        let mut groups = IndexMap::new();
+
+        for group in self.data.zulip_groups()?.values() {
+            let members = group.members().to_vec();
+            groups.insert(
+                group.name().to_string(),
+                v1::ZulipGroup {
+                    name: group.name().to_string(),
+                    members: members
```

Comment:
The list of members here is not sorted, which could result in unreproducible ordering for the members list (in the test for example it's not ordered).